docs/resolve/relationships.rst

```diff
@@ -142,50 +142,98 @@ This is by design because you wouldn't want one component to dispose the ``B`` o
 If you would rather control ``B`` disposal yourself all the time, :doc:`register B as ExternallyOwned() <../lifetime/disposal>`.
 
 
-Dynamic Instantiation (Func<B>)
+Dynamic Instantiation (`Func<B>`)
 -------------------------------
-Using an *auto-generated factory* can let you effectively call ``Resolve<T>()`` without tying your component to Autofac. Use this relationship type if you need to create more than one instance of a given service, or if you're not sure if you're going to need a service and want to make the decision at runtime. This relationship is also useful in cases like :doc:`WCF integration <../integration/wcf>` where you need to create a new service proxy after faulting the channel.
+Using an *auto-generated factory* can let you effectively call ``Resolve<B>()`` without tying your component to Autofac. Use this relationship type if you need to create more than one instance of a given service, or if you're not sure if you're going to need a service and want to make the decision at runtime. This relationship is also useful in cases like :doc:`WCF integration <../integration/wcf>` where you need to create a new service proxy after faulting the channel.
```

Comment:
Splitting existing suggestion across multiple lines

```suggestion
Using an *auto-generated factory* can let you resolve a new `B` programmatically within the control flow of your program, without requiring a direct dependency on the Autofac Library. Use this relationship type if you need to create more than one instance of a given service, or if you want to specifically control when the setup of the service occurs, or if you're not sure if you're going to need a service and want to make the decision at runtime. This relationship is also useful in cases like :doc:`WCF integration <../integration/wcf>` where you need to create a new service proxy after faulting the channel.
```

Reply:
This looks good. My slightly-nitpicky technical writing classes from college are kicking in, though, so some minor changes to make the bullets consistent and not use "Autofac Library" as a proper noun.

```suggestion
Using an *auto-generated factory* can let you resolve a new `B` programmatically within the control flow of your program, without requiring a direct dependency on the Autofac library. Use this relationship type if:

* You need to create more than one instance of a given service.
* You want to specifically control when the setup of the service occurs.
* You're not sure if you're going to need a service and want to make the decision at runtime.

This relationship is also useful in cases like :doc:`WCF integration <../integration/wcf>` where you need to create a new service proxy after faulting the channel.
```